str/helper/mt_extractor.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member
+"""
+This Hail Query script outputs a Hail matrix table  from a BGZIPPED VCF file.
+ analysis-runner --dataset "bioheart" \
+    --description "mt_extractor" \
+    --access-level "test" \
+    --output-dir "str/polymorphic_run/mt/v1" \
+    mt_extractor.py --file-path=gs://cpg-bioheart-test/str/polymorphic_run/bzip/v1/tester_file.vcf.bgz
+
+"""
+
+import hail as hl
+import click
+
+from cpg_utils.hail_batch import output_path, init_batch
+
+
+@click.option(
+    '--file-path',
+    help='GCS file path to BGZIP VCF file.',
+    type=str,
+)
+@click.command()
+def main(file_path):
+    """writes a BGZIP VCF as a Hail Matrix Table to a GCS bucket"""
+
+    init_batch()
+    gcs_path = output_path('str.mt', 'analysis')
```

Comment:
I'm not super wild about this, does all this data need to be in an analysis bucket? I'm mostly concerned that this script becomes a way of taking any VCF data and transcribing it to a less protected bucket

Reply:
removed analysis from output bucket